validator/specs/lib/validateI3dmSpec.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+var validateI3dm = require('../../lib/validateI3dm');
+
+describe('validateI3dm', function() {
+
+    it('returns true if the i3dm tile is valid, returns false if the i3dm has invalid magic', function() {
+        expect(validateI3dm(createInvalidMagic()).result).toBe(false);
+    });
+
+    it('returns true if the i3dm tile is valid, returns false if the i3dm has invalid version', function() {
+        expect(validateI3dm(createInvalidVersion()).result).toBe(false);
+    });
+
+    it('returns true if the i3dm tile is valid, returns false if the i3dm has wrong byteLength', function() {
+        expect(validateI3dm(createWrongByteLength()).result).toBe(false);
+    });
+
+    it('returns true if the i3dm tile is valid, returns false if the i3dm has invalid gITF Format', function() {
+        expect(validateI3dm(createInvalidGltfFormat()).result).toBe(false);
+    });
+
+    it('returns true if i3dm matches spec with glTF field of the body being a url', function() {
+        var validatorObject = validateI3dm(createI3dmTileGltfUrl());
+        var message = validatorObject.message;
+        expect(validatorObject.result && message.includes("url")).toBe(true);
+    });
+
+    it('returns true if i3dm matches spec with glTF field of the body being an embedded binary glTF', function() {
```

Comment:
The wording here can be simplified to something like:

`validates an i3dm with an embedded binary glTF`. Same with the url test.

Like above, it's fine not to check the message.